tests/x11/hexchat_ssl.pm

```diff
@@ -29,6 +39,7 @@ sub run {
 
     if (my $url = get_var("XCHAT_URL")) {
         x11_start_program("$name --url=$url", target_match => "$name-main-window");
+        irc_send_message($name);
```

Comment:
Since you added this line could you please offer the verification run for "XCHAT_URL"?

Reply:
@lilyeyes hold on please, I'll run it later.

Reply:
> Since you added this line could you please offer the verification run for "XCHAT_URL"?

Test failed when I added `XCHAT_URL` param,
I'll try to fix it later.

Reply:
@STARRY-S the failed case is expected, correct?

Reply:
@lilyeyes @rfan1 The test code has been updated and
test passed with or without the `XCHAT_URL` parameter.

Reply:
and here is the [verification run with XCHAT_URL](https://openqa.suse.de/tests/8367420#).